projects/js-packages/publicize-components/src/components/social-previews/twitter.js

```diff
@@ -14,14 +14,10 @@ function Twitter( { tweets } ) {
 }
 
 export default withSelect( ( select, { title, description, image, url, media } ) => {
-	const { getTweetTemplate, getTweetStorm, getShareMessage, isTweetStorm } =
-		select( 'jetpack/publicize' );
+	const { getTweetTemplate, getShareMessage } = select( 'jetpack/publicize' );
 
-	let tweets = [];
-	if ( isTweetStorm() ) {
-		tweets = getTweetStorm();
-	} else {
-		tweets.push( {
+	const tweets = [
```

Comment:
It's not good to return a new array on every select. This could be the reason why all these components, and thus the all the blocks, are constantly re-rendering on typing.